.github/workflows/gradle.yml

```
@@ -11,13 +11,15 @@ on:
     branches: [ main, development ]
   pull_request:
-    branches: [ main ]
+    branches: [ main, development ]
   workflow_dispatch:
 
 env:
   COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
 
 jobs:
+  # avoid double CI runs on push and PR, from https://github.com/orgs/community/discussions/57827
+  if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name != github.event.pull_request.base.repo.full_name
```

actionlint (1.7.4)

"steps" section is missing in job "if"

(syntax-check)